Gems/DiffuseProbeGrid/Code/Source/EditorComponents/EditorDiffuseProbeGridComponent.cpp

```diff
@@ -518,6 +518,17 @@ namespace AZ
                 return AZ::Edit::PropertyRefreshLevels::None;
             }
 
+            if (!m_controller.CanBakeTextures())
+            {
+                QMessageBox::information(
+                    QApplication::activeWindow(),
+                    "Diffuse Probe Grid",
+                    "Can't bake the textures. This function may require gpu raytracing support",
```

Comment:
We should drop the "may", it absolutely requires GPU raytracing. :) Suggested rewording:
```suggestion
                    "Can't bake the textures. Diffuse probe calculations require GPU raytracing support",
```